sw/host/opentitantool/src/command/certificate.rs

```diff
@@ -19,6 +19,8 @@ use ot_certs::{codegen, template, x509};
 pub enum CertificateCommand {
     /// Generate a certificate template.
     GenTemplate(GenTplCommand),
+    /// Parse a certificate.
+    Parse(ParseCertificate),
```

Comment:
Unless I am mistaken, this code already does that since it's the name of the enum (`Parse`) and not of the structure that counts. I can use `opentitantool certificate parse` on my computer without any issue.